stdlib/3.4/enum.pyi

```diff
@@ -20,8 +21,6 @@ class Enum:
 class IntEnum(int, Enum):
     value = ...  # type: int
 
-_T = TypeVar('_T')
-
 def unique(enumeration: _T) -> _T: ...
```

Comment:
Interesting, so it works differently when used as a decorator. That seems like a mypy bug.